docs/basic-docs/atomicdex/atomicdex-tutorials/coins-file-update.md

```diff
@@ -0,0 +1,96 @@
+# How to update the coins file to work with the latest atomicDEX-API
+
+## Instructions
+
+- Compile the latest `mm2` binary following [these instructions](../atomicdex-setup/get-started-atomicdex.html)
+- Place the `coins` file in the old format in the same directory as the compiled `mm2` binary
```

Comment:
I think it is not a necessary, instead I suggest to mention that the user need to specify the first arg - any path to the source file, second arg - any path to the result file.

Thanks!

Reply:
It is guide for end users, while MM2 can actually accept any path as input and output it is better to be as explicit as possible in such cases.

Reply:
I am going to start documenting all the cli params of mm2. That doc will mention that mm2 can accept any valid path for the arguments. I will also add a note that mentions it in this doc.